zookeeper/files/default/example.tmpl

```diff
@@ -0,0 +1,6 @@
+########################################################################
+# File managed by Salt at <{{ source }}>.
+# Your changes will be overwritten.
+########################################################################
+
+This is an example file from SaltStack template-formula.
```

Comment:
Replaces instances of "template-formula" with "zookeeper-formula" where that makes sense.  i.e. in places where name of this formula is implied.